### PR TITLE
Add dag that clears file system caches on scheduler and workers.

### PR DESCRIPTION
**Pull request by anitakar**
This dag allows to decrease memory usage of worker and scheduler pods on k8s.

## Description

Fixes #<ISSUE-NUMBER>

Note: It's a good idea to open an issue first for discussion.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [x] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [X] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

**Comment by leahecole** [comment]
Hey Anita! Thank you for your DAG! This looks awesome. I have a few requests about the PR as a whole

1. What do you think about having the README info be in the Cloud Docs? It might be more accessible to customers there. Pavel and/or I could help migrate info about it over (if that's what you want and you'd like to pair, please feel free to put time on my calendar
2. Regardless of location, consider including visuals or more descriptive indicators of when a user would want to use this
3. Are there any consequences to using this dag? List those risks
4. Are there any unsupported use cases for using this dag? Let users know.
2. For reasons relating to this repo infrastructure, this dag needs to be in `composer/workflows`. Someday I hope to move it and the airflow db cleanup dag to tools, but I'm not ready for that yet. Because of that move, if we don't put the README info in the cloud docs, it should be incorporated into a docstring comment (see [airflow_db_cleanup example](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/4ba588b5a48594411fd4e86abd824b28821bd592/composer/workflows/airflow_db_cleanup.py#L16-L48)) 
3. Please add a unit test for this DAG - it will be very simple. Since you're not setting any Airflow variables, it will likely be identical to this [simple_test](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/composer/workflows/simple_test.py). It should be named `clear_file_system_caches_dag_test.py`. 
4. Please resolve the header-check bot errors by adding the [Apache license header found at this internal link](https://g3doc.corp.google.com/company/teams/opensource/releasing/preparing.md?cl=head#Apache-header)
5. Optional, but nice, If you join the `GoogleCloudPlatform` org on go/github, I believe the system tests will run for you automatically. Otherwise they'll stay stuck in this "Expected" state until I or another repo owner forces the run. (You can run tests locally following the authoring guide instructions)

**Review by leahecole — CHANGES_REQUESTED** [review]
Hey Anita! Thank you for your DAG! This looks awesome. I have a few requests about the PR as a whole

1. What do you think about having the README info be in the Cloud Docs? It might be more accessible to customers there. Pavel and/or I could help migrate info about it over (if that's what you want and you'd like to pair, please feel free to put time on my calendar
2. Regardless of location, consider including visuals or more descriptive indicators of when a user would want to use this
3. Are there any consequences to using this dag? List those risks
4. Are there any unsupported use cases for using this dag? Let users know.
2. For reasons relating to this repo infrastructure, this dag needs to be in `composer/workflows`. Someday I hope to move it and the airflow db cleanup dag to tools, but I'm not ready for that yet. Because of that move, if we don't put the README info in the cloud docs, it should be incorporated into a docstring comment (see [airflow_db_cleanup example](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/4ba588b5a48594411fd4e86abd824b28821bd592/composer/workflows/airflow_db_cleanup.py#L16-L48)) 
3. Please add a unit test for this DAG - it will be very simple. Since you're not setting any Airflow variables, it will likely be identical to this [simple_test](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/composer/workflows/simple_test.py). It should be named `clear_file_system_caches_dag_test.py`. 
4. Please resolve the header-check bot errors by adding the [Apache license header found at this internal link](https://g3doc.corp.google.com/company/teams/opensource/releasing/preparing.md?cl=head#Apache-header)
5. Optional, but nice, If you join the `GoogleCloudPlatform` org on go/github, I believe the system tests will run for you automatically. Otherwise they'll stay stuck in this "Expected" state until I or another repo owner forces the run. (You can run tests locally following the authoring guide instructions)

File: composer/clear_file_system_caches_dag_test.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2018 Google LLC
```
Comment (leahecole) [inline composer/clear_file_system_caches_dag_test.py:1]:
```suggestion
# Copyright 2021 Google LLC
```

Comment (leahecole) [inline composer/clear_file_system_caches_dag_test.py:1]: Needs to be in same directory as the dag

**Review by leahecole — APPROVED** [review]

**Review by tswast — APPROVED** [review]
Cool! Today I learned about `/proc/sys/vm/drop_caches`. Folks on 
 https://unix.stackexchange.com/q/17936/11193 seem to agree this is the way to do it.

**Pull request merged**